src/include/execution/ast/builtins.h

```diff
@@ -7,6 +7,16 @@ namespace terrier::execution::ast {
 // The list of all builtin functions
 // Args: internal name, function name
 #define BUILTINS_LIST(F)                                                \
+  /* SQL NULL. */                                                       \
+  F(IsNull, isNull)                                                     \
+  F(IsNotNull, isNotNull)                                               \
+  F(NullBool, nullBool)                                                 \
+  F(NullInt, nullInt)                                                   \
+  F(NullReal, nullReal)                                                 \
+  F(NullDecimal, nullDecimal)                                           \
+  F(NullString, nullString)                                             \
+  F(NullDate, nullDate)                                                 \
+  F(NullTimestamp, nullTimestamp)                                       \
```

Comment:
Can you modify these value creation builtins to indicate that they're SQL values with NULL? Similar to the `intToSql` conversion calls below.